packages/node/test/setup/set-env-vars.ts

```diff
@@ -0,0 +1 @@
+process.env.MASTER_KEY_MNEMONIC = 'achieve climb couple wait accident symbol spy blouse reduce foil echo label';
```

Comment:
This file gets run when tests start (so we don't need to set it everywhere)

Reply:
It was looking awkward :pray: